python/CONTRIBUTING.rst

```diff
@@ -77,10 +77,8 @@ Ready to contribute? Here's how to set up `nessie` for local development.
    Now you can make your changes locally.
 
 5. When you're done making changes, check that your changes pass flake8 and the
-   tests, including testing other Python versions with tox::
+   tests, including testing other Python versions with tox and all docs have been generated::
 
-    $ flake8 pynessie tests
-    $ python setup.py test or pytest
     $ tox
 
    To get flake8 and tox, just pip install them into your virtualenv.
```

Comment:
Do we need to mention `flake8` now? I guess `tox` will take care of it automatically?

Reply:
I think it could be the case you want to just run `flake8` alone though without tox 🤔

Reply:
In my experience, the standalone `flake8` results do not always match what it detects under `tox` :shrug:

Reply:
Weird, I have been always using the standalone `flake8`, maybe because in `tox` in needs to run both for `pynessie` and `tests` folders? 🤔

Reply:
IDK, I hit a couple of cases when the `tox` run of `flake8` was stricter than a direct run :shrug:

Reply:
> IDK, I hit a couple of cases when the `tox` run of `flake8` was stricter than a direct run 🤷

Ah maybe because on `tox -e flake8`, it runs as well `mypy` which Python`s type checker as you can see [here](https://github.com/projectnessie/nessie/blob/main/python/tox.ini#L34). I think should be separated into different tox step for clarity 🤔